site/content/getting-started.md

```diff
@@ -183,3 +183,7 @@ In order to start up the instance again:
 ~~~sh
 ~/hono/example/target/hono$ docker stack deploy -c docker-compose.yml hono
 ~~~
+
+## View metrics
+
+Open [http://localhost:3000/dashboard/db/hono?orgId=1]() in a browser. Login is admin/admin.
```

Comment:
You should put the URL between the rounded brackets and put a descriptive text between the square brackets. We usually format literals monospaced, i.e. `admin/admin`